### DISCUSSION TITLE
Can't pay for Github Copilot

### DISCUSSION THREAD

**Discussion by meandrewdev** [community/community#33430] [Copilot Conversations] [answered]
I added my card, but did not notice that the limit for online purchases was used, so attempts to withdraw money was failed. I increased the limit, but now I can't pay because it won't let me renew my payment card details. What shall I do?

**Comment by samuel-ma** [comment]
@meandrewdev 
Pretty simple fix, just go to your settings--->Billing and plans---->Update Payment method
That should fix your problem

**Reply by meandrewdev** [reply]
Yes, I have already done this several times, moreover, I even reissued the card and tried to enter a new number, but I still get this message:
`Your account is currently locked from purchases. Please update your payment information. `
After pressing the button `Get access to GitHub Copilot->Submit`

**Reply by yuriidiiego** [reply]
I have the same problem, so far I haven't found a solution.

**Reply by froxCZ** [reply]
I have the same issue, even if I remove the card and add a new one, it says Your account is currently locked from purchases. Please update your payment information.

**Reply by jinek** [reply]
+1

**Reply by sakshamsri4** [reply]
+1

**Reply by jazkw** [reply]
+1

**Reply by NgxDev** [reply]
This is not a fix, unfortunately. I need to do this every month. No problem with my credit card. I edit and just write in all the same details again, hit Save and after that I get billed just fine. And then, next month, the same... billing fails, I edit, just re-enter the same credit card info, and then I get billed successfully. We shouldn't need to do this every month, wth?

**Reply by DeepakKapiswe** [reply]
same problem here

**Reply by davecheney** [reply]
@DeepakKapiswe it looks like your card was declined. We cannot assist you with billing issues via this forum. Please contact support https://support.github.com/contact who can assist you.

**Comment by iamcjwilliams** [comment, marked as answer]
@meandrewdev The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)

**Reply by meandrewdev** [reply]
I already did this 5 days ago, but ticket #1800384 is still unanswered. It's a strange situation, I want to buy a service, but they don't let me do it :)

**Comment by 3kh0** [comment]
You might want to go to your bank with this one

**Reply by meandrewdev** [reply]
I pay for other services normally, it's not the bank's fault

**Comment by theindianappguy** [comment]
i faced the same even after updating the payment method it shows i am locked

**Reply by iamcjwilliams** [reply]
@theindianappguy Have you opened a ticket on our [Support page](https://support.github.com/contact)?

**Comment by cognivore** [comment]
Doesn't work while trying to pay for the whole organization using Wise bank's debit card (ex Transferwise).

Please help.

**Reply by cognivore** [reply]
@iamcjwilliams sorry for being annoying but surely you have some information about making Wise business payments. We tried many times and it never worked. Should we buy American Dollars?

**Reply by iamcjwilliams** [reply]
@cognivore Have you opened a support ticket? If not, please visit [Support page](https://support.github.com/contact). They should be able to help you get this sorted

**Reply by cognivore** [reply]
@iamcjwilliams sure, but how will they help. Surely it's a known issue with github not being able to process debit cards of Wise bank. https://support.github.com/ticket/personal/0/2108746

**Reply by lucasriechelmann** [reply]
@cognivore I think it is a problem with international Banks. My trial ended and I am trying to use Revolut or Paypal and it is not possible to pay.

**Reply by k6xiao** [reply]
Your account is currently locked from purchases. Please update your payment information. 使用paypal，而且paypal余额充足，无论怎么做，都始终提示这个。

**Comment by ayushagwl499** [comment]
I removed my existing paypal account and added it again and it seemed to solve the issue
I was getting the same error earlier with debit cards

**Comment by cognivore** [comment]
But we don't have a paypal account... We're using a simple debit card from
a Belgian bank to pay for copilot for the whole organization but the
payment gets declined every single time. Honestly, it's ridiculous by now!

**Comment by patil-pratik-87** [comment]
The payment keeps failing for me too in India. I tried it with different bank credit cards I have, same error everywhere. 
Bank Officials tell me its merchants fault.

**Comment by r3flow** [comment]
Today I received an email that the billing attempt was unsuccessful. I registered months ago with a Revolut card. There is nothing in Revolut's transaction history. It does not show any attempts(!) of payment either. In previous months, GitHub successfully deducted the monthly fee from the revolut card. The card is not frozen and the corresponding balance is available. The problem must be on the GitHub side, because the revolut has not even received the payment attempt.

**Comment by jinek** [comment]
I understand support would do better support, but I see other people having similar problems. I'm trying to pay with card of Indian Overseas Bank. It mostly fails but sometimes success. I couldn't understand why it's different. 

> The credit card was declined by the payment processor, please verify the card's information. If the problem persists, please contact the card's issuer.

Also just found out that sometimes it fails before 3DS page, but sometimes after.

**Comment by nulevii** [comment]
I think there is some problem with my acc, because I can't pay using cards from different banks

**Comment by lucas-krawczyk** [comment]
Go to the Billing info, remove the card, and add it again. Worked for me.

**Reply by trytryty1** [reply]
It wont let me add the card again

**Reply by illarion** [reply]
It works for me, but I have to do it each month :(

**Reply by dominicarrojado** [reply]
Doesn't work for me. How long will it take to try and bill again?

**Reply by SharbelOkzan** [reply]
Removing and then readding the same card is working for me. It's annoying to do it each month though

**Comment by siccovansas** [comment]
If you're using a debit card (in the EU), try to set the card to world/international payments.

**Comment by Manpreet-Singh-MS** [comment]
Update Payment Method

**Comment by arun-ku** [comment]
Indian cards do not allow international subscriptions due to security issues. I think that is the issue. I have tried multiple cards but still no luck.

**Comment by karan-bohra** [comment]
Github needs to improve its billing section, it is the worst!

**Comment by ElenaGera** [comment]
I had the same problem and even changing payment details didn't work. But I resolved the issue in the next way:
1. Go to the setting 
2. Click on the "Copilot" left-side menu item
3. There should be a "Buy copilot" banner
It looks like when your payment fails, your subscription has ended and you need to go through the “purchase” procedure for the subscription again. Click “buy”, the next step should be the payment, and !voila! everything is working
